hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcConnection.java

```diff
@@ -258,6 +252,8 @@ private void connect() {
         .option(ChannelOption.SO_KEEPALIVE, rpcClient.tcpKeepAlive)
         .option(ChannelOption.CONNECT_TIMEOUT_MILLIS, rpcClient.connectTO)
         .handler(new BufferCallBeforeInitHandler()).localAddress(rpcClient.localAddr)
+        .option(ChannelOption.WRITE_BUFFER_WATER_MARK,
+            ((NettyRpcClient) rpcClient).writeBufferWaterMark)
```

Comment:
This cast is redundant.

Reply:
Good catch, no longer necessary, from an earlier iteration.